### ISSUE TITLE
Porting to Kodi 16.0 “Jarvis"

### ISSUE THREAD

**Issue by bigretromike** [bigretromike/nakamoriplugin#63]
Jan.29 - RC2
Jan.21 - RC1

Its time to start porting time :-)

**Comment by bigretromike** [comment]
- [x] Install

**Comment by bigretromike** [comment]
- [x] Start

**Comment by bigretromike** [comment]
- [x] List items

**Comment by bigretromike** [comment]
- [x] Mark watched
- [x] Vote ep/season

**Comment by bigretromike** [comment]
- [x] Playing file

**Comment by bigretromike** [comment]
- [ ] After playing : ERROR: Exception caught on main loop. Exiting

**Comment by bigretromike** [comment]
```
WARNING: RunScript called for a non-script addon 'plugin.video.nakamoriplugin'. This behaviour is deprecated.

ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.nakamoriplugin/?ep_id=11578%3furl%3durl&file=http%3a%2f%2f10.1.1.100%3a8112%2fvideolocal%2f9395%2ffile.mkv&filename=4&mode=7&name=-continue-&offset=11&poster_file=3&url=http%3a%2f%2f10.1.1.100%3a8111%2fJMMServerKodi%2fGetMetadata%2f0%2f7%2f9384

ERROR: CDVDMsgGeneralSynchronize - timeout

ERROR: Got MSGQ_ABORT or MSGO_IS_ERROR return true

ERROR: Control 50 in window 10025 has been asked to focus, but it can't
WARNING: RunScript called for a non-script addon 'plugin.video.nakamoriplugin'. This behaviour is deprecated.
```
Error look similar to Android problem: 
https://github.com/bigretromike/nakamoriplugin/issues/54

**Comment by bigretromike** [comment]
- [x] ERROR: XFILE::CDirectory::GetDirectory

**Comment by bigretromike** [comment]
fixed in 1.2.8
